CONTRIBUTING.md

```diff
@@ -30,6 +30,26 @@ We use GitHub templates to manage the requests. You will either select:
 
 Ensure the appropriate label is selected before submitting your issue.
 
+## Reviewing Pull Requests
+
+Each pull request requires at least *one* review and approval from an active
+contributor. The primary function of reviews is to ensure code contributors
+are not blocked.
```

Comment:
IMO, the primary function of reviews is to ensure quality. However, we also want PRs to be reviewed quickly, to avoid the blocking problem.

Reply:
Agree with @ml8mr